### ISSUE TITLE
Different type format for content type

### ISSUE THREAD

**Issue by Gonzalo2683** [minnur/Alter-Entity-Autocomplete#3]
Hi, is it possible to have different types of auto-completed formats for each type of content?

**Comment by minnur** [comment]
yes, you would have to add some more custom code to this file https://github.com/minnur/Alter-Entity-Autocomplete/blob/master/src/EntityAutocompleteMatcher.php

**Comment by minnur** [comment]
@Gonzalo2683 https://downloads.minnur.com/drupal/entity-reference-patterns
